File: sumadedosnumeros/app/src/main/java/com/example/sumadedosnumeros/MainActivity.java
package com.example.sumadedosnumeros;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity implements View.OnClickListener{
    EditText et1;
    EditText et2;
    TextView tv1;
    Button btnSumar;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        et1 = (EditText)findViewById(R.id.txtNumero1);
        et2 = (EditText)findViewById(R.id.txtNumero2);
        tv1 = (TextView)findViewById(R.id.lblResultado);
        btnSumar = (Button)findViewById(R.id.btnSumar);
        btnSumar.setOnClickListener(this);

    }

    @Override
    public void onClick(View v)
    Toast.makeText(getApplicationContext(), "a presionado el boton", Toast.LENGTH_SHORT).show(){
    }
}

File: Examen11/app/src/main/java/com/example/examen11/MainActivity.kt
package com.example.examen11

import android.annotation.SuppressLint
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val boton1 = findViewById<Button>(R.id.boton1)
        boton1.setOnClickListener {
            val intent: Intent = Intent(this, MENU::class.java)
            startActivity(intent)
        }

        val boton2 =findViewById<Button>(R.id.boton2)
        boton2.setOnClickListener{
            finish()

        }
    }
}